website/docs/hms.md

```diff
@@ -127,6 +127,16 @@ cd $HUDI_HOME/hudi-sync/hudi-hive-sync
 --table <tableName>
 ```
 
+```shell md title="example for s3 with HMS"
```

Comment:
Let's keep move this up and move the JDBC command below the HMS one. For the jdbc one, we can call out it's the old way of syncing to hive.

Reply:
done